### PR TITLE
Gjw/54 add about dialog

### PR DESCRIPTION
**Pull request by gjwgit**
# Pull Request Details

## What issue does this PR address

- Adds an about dialog.

- Link to associated issue: https://github.com/anusii/keypod/issues/54

## Checklist

[Use the check-list below to ensure your branch is ready for PR]

- [x] Changes adhere to the team style and coding guideline
- [x] The one line summary included in CHANGELOG.md
- [x] No confidential information
- [x] No duplicated content
- [ ] No lint errors (`make prep` or `flutter analyze lib`
                       python: use pre-commit ruff hook to check on commit)
- [x] No lint check errors related to your changes
- [x] Tested on at least one device
     - [x] Android
     - [ ] Chrome
     - [ ] iOS
     - [x] Linux
     - [ ] MacOS
     - [ ] Windows
- [x] Added 2 reviewers

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

## Finalising

Once PR discussion is complete and 2 reviewers have approved:

- [x] Merge dev into the branch
- [x] Resolve any conflicts
- [x] Bump appropriate version number
- [ ] Merge PR into dev

### CODE REVIEW

File: lib/screens/home.dart [hunk 1]
```diff
@@ -69,6 +69,36 @@ class HomeState extends State<Home> with SingleTickerProviderStateMixin {
...
         centerTitle: true,
         title: Text(widget.appName),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.info),
+            onPressed: () {
+              showAboutDialog(
+                context: context,
+                applicationName: 'Key Pod',
+                applicationVersion: '1.2.3',
```
Comment (cdawei) [inline lib/screens/home.dart:79]: This version number is different from the one in `pubspec.yaml`, is this intentional?

**Review by cdawei — APPROVED** [review]
Worked in my test.

File: lib/screens/home.dart (hunk 1 above)
Comment (gjwgit) [reply lib/screens/home.dart:79]: Yes, I need some code to get the actual version number - a task for another day.

**Review by junhaow1 — APPROVED** [review]
Works on my machine

File: lib/screens/home.dart (hunk 1 above)
Comment (cdawei) [reply lib/screens/home.dart]: We have a [function in solidpod](https://github.com/anusii/solidpod/blob/494d372fbea2e3e29835df34da7b98335cb40da6/lib/src/solid/api/rest_api.dart#L626) to do this. Just did that here.

Comment (gjwgit) [reply lib/screens/home.dart]: Excellent. Thanks for adding that in @cdawei.

**Pull request merged**